docs/src/gui/gmoccapy.adoc

```diff
@@ -1037,8 +1037,9 @@ Modify your INI File to include the following:
 [source,{ini}]
 ----
 [RS274NGC]
-# Enables the reading of INI and HAL values from G-code
-FEATURES=12
+# Make sure INI_VARS and HAL_PIN_VARS are not set to 0. They are set to 1 by default.
+# LinuxCNC versions below 2.8 require the following line to be uncommented:
+# FEATURES = 12
```

Comment:
I would word it a bit differently to make it more clear

```suggestion
# These two values doesn't have to be set explicitly because their default value is 1. Only make sure they are not overwritten in this case.
# In LinuxCNC versions prior to 2.8 these settings were set with the FEATURES bitmask (FEATURES = 12)
INI_VARS = 1
HAL_PIN_VARS = 1
```

And I would suggest to target this PR to 2.8 and merge it up to master.

Reply:
Or maybe remove these lines completely and add your lines as a note below the code block:
```
[NOTE]
Make sure INI_VARS and HAL_PIN_VARS are not set to 0. They are set to 1 by default. 
```
The hint for versions < 2.8 I would remove, as it is more confusing than helpful.